src/lib/html.ts

```diff
@@ -49,374 +43,177 @@ const convert_to_id = (header: string): string => {
  * @returns results in XML format
  */
 function create_impl_list(impl: Implementer[]): string {
-    const root: any = {
-        section : {
-            h2 : {
-                $ : {
-                    id : "sec-implementer-list",
-                },                    
-                _ : "List of Implementations",
-            },
-            ol : {
-            },
-        },
-    }
+    const dom: DocumentFragment = JSDOM.fragment('<section id="sec-implementer-list"><h2>List of Implementations</h2></section>');
+    const section: HTMLElement = dom.querySelector('section');
 
-    // The value of li is an array, ie, a repeated set if li elements...
-    root.section.ol.li = impl.map((implementer: Implementer): any => {
-        // Some implementation reports may not provide a website reference...
-        const name = 'variant' in implementer ? `${implementer.name} (${implementer.variant})` : implementer.name;
-        if ('ref' in implementer) {
-            return {
-                a : {
-                    $ : {
-                        href : implementer.ref,
-                    },
-                    _ : name,
-                },    
-            }
-        } else {
-            return name
-        }
-    }); 
+    const ol = add_child(section, 'ol');
 
-    // This is where the object is turned into an XML serialization...
-    return builder.buildObject(root)
+    for (const implementer of impl) {
+        const li = add_child(ol, 'li');
+        const name = 'ref' in implementer ? `<a href=${implementer.ref}>${implementer.name}</a>` : `${implementer.name}`;
```

Comment:
```suggestion
        const name = 'ref' in implementer ? `<a href="${implementer.ref}">${implementer.name}</a>` : `${implementer.name}`;
```

Reply:
Also, note (as I'm sure you know) that constructing HTML like this is not generally considered safe, but given the scope of this script I think it's OK.